### ISSUE TITLE
Invoking the venv python shouldn't require knowing the path

### ISSUE THREAD

**Issue by polotek** [polotek/docker-web-dev-environment#4]
The app is configured to run in a virtualenv. But I decided not to activate the virtual env in the environment because that seemed like an unstable requirement to make sure things work. Instead invoke the commands directly via the path. I tried to use the env variable `PYTHON_BIN_PATH` to get around knowing the actual path, so it could be configurable in the future. But I couldn't get that to work for some reason.

In short, come up with a better way to invoke the virtualenv commands while keeping the location of the virtualenv itself configurable.
